cmd/deploy.go

```diff
@@ -12,13 +12,27 @@ import (
 )
 
 var (
+	deployExample = `
+# Deployment you would like to deploy to Airflow cluster:
+astro deploy physical-diameter-1566
```

Comment:
```suggestion
$ astro deploy physical-diameter-1566
```

To differentiate commands run from output/response?

(and the next `astro deploy`)

Reply:
@ashb fixed.